test/_routes.js

```diff
@@ -90,7 +90,7 @@ module.exports = function () {
         .get(`${openapi.defaultRoutePrefix}.json`)
         .expect(200, (err, res) => {
           assert(!err, err)
-          assert.strictEqual(Object.keys((res.body.paths))[0], '/{id}/')
+          assert.strictEqual(Object.keys((res.body.paths))[0], '(?:/([^/]+?))/')
```

Comment:
Not sure why this test was complaining for me, but when I run `npm test` on `main`, it was failing. By updating it to this regex, it was passing for me again.